src/templates/header.html

```diff
@@ -1,79 +1,60 @@
-                <img class="usa-banner-header-flag" src="{{ uswds.path }}img/favicons/favicon-57.png" alt="U.S. flag">
+
+<section class="usa-banner">
+    <div class="usa-accordion">
+        <header class="usa-banner__header">
+            <div class="usa-banner__inner">
+                <div class="grid-col-auto">
+                    <img class="usa-banner__header-flag" src="{{ uswds.path }}/img/us_flag_small.png" alt="U.S. flag">
+                </div>
+                <div class="grid-col-fill tablet:grid-col-auto">
+                    <p class="usa-banner__header-text">An official website of the United States government</p>
+                    <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
+                </div>
+                <button class="usa-accordion__button usa-banner__button"
+                        aria-expanded="false" aria-controls="gov-banner">
+                    <span class="usa-banner__button-text">Here’s how you know</span>
+                </button>
             </div>
-            <div class="grid-col-fill tablet:grid-col-auto">
-                <p class="usa-banner-header-text">An official website of the United States government</p>
+        </header>
+        <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+            <div class="grid-row grid-gap-lg">
+                <div class="usa-banner__guidance tablet:grid-col-6">
+                    <img class="usa-banner__icon usa-media-block__img" src="{{ uswds.path }}/img/icon-dot-gov.svg"
+                         alt="Dot gov">
+                    <div class="usa-media-block__body">
+                        <p>
+                            <strong>The .gov means it’s official.</strong>
+                            <br>
+                            Federal government websites often end in .gov or .mil. Before sharing sensitive information,
+                            make sure you’re on a federal government site.
+                        </p>
+                    </div>
+                </div>
+                <div class="usa-banner__guidance tablet:grid-col-6">
+                    <img class="usa-banner__icon usa-media-block__img" src="{{ uswds.path }}/img/icon-https.svg"
+                         alt="Https">
+                    <div class="usa-media-block__body">
+                        <p>
+                            <strong>The site is secure.</strong>
+                            <br>
+                            The <strong>https://</strong> ensures that you are connecting to the official website and
+                            that any information you provide is encrypted and transmitted securely.
+                        </p>
+                    </div>
+                </div>
             </div>
         </div>
-    </header>
-<div class="usa-overlay"></div>
+    </div>
+</section>
+
 <header class="usa-header usa-header-extended usgs-header" role="banner">
-   <div class="usa-navbar">
```

Comment:
I think this should be put back in. This is an example template and it's showing how to add navigation if desired to the header.

Reply:
Do you mean put back all the navigation that was in the USA the sample header? The, perhaps excessively, strict guidelines for the USGS header won't allow any links and only let us show the USGS specific search box, which does not always return the page specific results users would expect (which can also be hidden). I can put the links back, but I don't think we will ever be allowed to use them.

Reply:
yes. Pubs Warehouse has this kind of navigation (see https://pubs.er.usgs.gov/).

Reply:
Okay, gotcha. The way it is set up for Pubs Warehouse is different than the way it was originally set up here.